app/modules/ia_config_reader.py

```diff
@@ -194,6 +211,6 @@ def get_detect_model_frontend_data(self):
         return result
 
 
-ic = IaConfig()
+# ic = IaConfig()
```

Comment:
and in the constructor, if the code is finished with, it should be removed, not commented out

Reply:
Removed